### DISCUSSION TITLE
Is it possible to integrate IDOM for frontend with atrilabs?

### DISCUSSION THREAD

**Discussion by takan1** [Atri-Labs/atrilabs-engine#661] [Help: Miscellaneous]
Hi I am very new here. For my next project, I wanted to use [IDOM](https://github.com/idom-team/idom) so that I can code frontend in Python. Would it be possible to integrate it on atrilabs framework so that I can leverage visual help from atrilabs?

**Comment by cruxcode** [comment]
@takan1 Thanks for pointing me toward IDOM. I am taking a look at it.
